### ISSUE TITLE
Location based Outlets

### ISSUE THREAD

**Issue by lchristus** [hpi-swa-teaching/MensaWidget#29]
**As a user** I want to choose between different outlet locations to learn about local offerings when I am in a different area.

**Comment by OscarUlb** [comment]
I would rephrase the story to make it clear that we are not switching individual cafeterias but rather the user's location. My idea was: 
**As a user**, I want to change my location and see the nearest outlets **to** view local offerings when I am in a different area.
